packages/terra-layout/src/LayoutSlidePanel.module.scss

```diff
@@ -39,45 +39,47 @@
   }
 
   .panel.is-tiny {
-    width: var(--layout-slide-panel-phone-panel-width, calc(100% - 3.571rem));
+    width: var(--terra-layout-slide-panel-phone-panel-width, calc(100% - 3.571rem));
   }
 
   .panel.is-small {
-    width: var(--layout-slide-panel-tablet-panel-width, 22.857rem);
+    width: var(--terra-layout-slide-panel-tablet-panel-width, 22.857rem);
   }
 
   // Overlay
   .is-open.is-overlay > .panel {
     /* stylelint-disable selector-max-compound-selectors */
     [dir=ltr] & {
-      box-shadow: var(--layout-slide-panel-overlay-panel-box-shadow-ltr, 1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8));
+      box-shadow: var(--terra-layout-slide-panel-overlay-panel-ltr-box-shadow, 1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8));
     }
 
     [dir=rtl] & {
-      box-shadow: var(--layout-slide-panel-overlay-panel-box-shadow-rtl, -1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8)); 
+      box-shadow: var(--terra-layout-slide-panel-overlay-panel-rtl-box-shadow, -1px 0 0 0 #1c1f21, 0 0 5px 1px #000, 0 0 12px 2px rgba(0, 0, 0, 0.8));
     }
     /* stylelint-enable selector-max-compound-selectors */
   }
 
   // Squish
   .is-open.is-squish {
     > .content {
-      width: calc(100% - #{var(--layout-slide-panel-desktop-panel-width, 18.571rem)} - #{var(--layout-slide-panel-squish-panel-border-right-width, 1px)});
+      width: calc(100% - #{var(--terra-layout-slide-panel-desktop-panel-width, 18.571rem)} - #{var(--terra-layout-slide-panel-squish-panel-border-right, 1px)});
     }
 
     > .panel {
-      background-color: var(--layout-slide-panel-squish-panel-background-color, #fff);
-      border-right: var(--layout-slide-panel-squish-panel-border-right-width, 1px) solid var(--layout-slide-panel-squish-panel-border-color, #d3d4d5);
+      background-color: var(--terra-layout-slide-panel-squish-panel-background-color, #fff);
+      border-right-color: var(--terra-layout-slide-panel-squish-panel-border-right-color, #d3d4d5);
+      border-right-style: solid;
+      border-right-width: var(--terra-layout-slide-panel-squish-panel-border-right-width, 1px);
       transition: none;
-      width: calc(#{var(--layout-slide-panel-squish-panel-border-right-width, 1px)} + #{var(--layout-slide-panel-desktop-panel-width, 18.571rem)});
+      width: calc(#{var(--terra-layout-slide-panel-squish-panel-border-right, 1px)} + #{var(--terra-layout-slide-panel-desktop-panel-width, 18.571rem)});
```

Comment:
Left a note about removing desktop on some code above these lines.

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/429/commits/82c3fd5b63ab11a574da1f8323fde4731425153b